### DISCUSSION TITLE
Configuração de e-mail inserida errada, como deletar?

### DISCUSSION THREAD

**Discussion by maxcanoas** [frontendbr/forum#2393] [Perguntas e Respostas] [answered]
boa tarde, eu estava configurando o git aqui no meu notebook, e errei um dos itens, e ao invés de colocar email, coloquei apenas mail, gostaria de saber se tem como eu deletar essa informação?

![image](https://user-images.githubusercontent.com/39919185/213888416-9246a87b-6fcd-4cb1-a11c-7e50eb524b79.png)

**Comment by gstcarv** [comment, marked as answer]
Tente:

```git config --global --unset user.mail```

**Reply by maxcanoas** [reply]
Ah, excelente, funcionou.
Valeu, cara.
